src/main/resources/pageviews_schema.avro

```diff
@@ -13,13 +13,13 @@
                 {"name": "userid", "type": {
                     "type": "string",
                     "arg.properties": {
-                        "regex": "User_[1-9]{0,1}"
+                        "regex": "User_[1-9]"
                     }
                 }},
                 {"name": "pageid", "type": {
                     "type": "string",
                     "arg.properties": {
-                        "regex": "Page_[1-9][0-9]?"
+                        "regex": "Page_[1-9][0-9]"
```

Comment:
Looks like Page1 - Page9 will not be possible anymore. Is it fine?

Reply:
@rajdangwal good catch.  IMO all changes to _existing_ schema definitions should be backward compatible and generate the same range of data.

Reply:
@rajdangwal Page_1 -Page_9 was not generated by this schema earlier also. It was always in this range - Page_10 - Page_99.

Reply:
Yeah, true. Also, I never saw User_ which is possible with this regex `User_[1-9]{0,1}`.
So, this is backward compatible change then. 😄